### PR TITLE
added import dialog for authorization resources

### PR DESCRIPTION
**Pull request by edewit**
added only [the screen](https://marvelapp.com/prototype/78j1b69/screen/82976116) without the import logic

### CODE REVIEW

**Review by ssilvert — DISMISSED** [review]
This breaks the partial import tests.

File: src/clients/authorization/ImportDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+                label={t("common:on")}
+                labelOff={t("common:off")}
+                isChecked={imported.allowRemoteResourceManagement}
+                isDisabled
+              />
+            </FormGroup>
+          </Form>
+          <div className="pf-u-mt-md">
+            {Object.entries(imported)
+              .filter((entry) => Array.isArray(entry[1]))
```
Comment (jonkoops) [inline src/clients/authorization/ImportDialog.tsx:133]: You could also destructure this.
```suggestion
              .filter(([, value]) => Array.isArray(value))
```

File: src/clients/authorization/ImportDialog.tsx [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+  onConfirm: (value: ResourceServerRepresentation) => void;
+  closeDialog: () => void;
+};
+
+export const ImportDialog = ({ onConfirm, closeDialog }: ImportDialogProps) => {
+  const { t } = useTranslation("clients");
+  const [imported, setImported] = useState<ResourceServerRepresentation>({});
+  return (
+    <Modal
+      title="Import"
```
Comment (jonkoops) [inline src/clients/authorization/ImportDialog.tsx:29]: This title will have to be translated.

**Review by jonkoops — CHANGES_REQUESTED** [review]

**Review by jonkoops — APPROVED** [review]

**Comment by jonkoops** [comment]
@edewit Looks good to me, but there are some merge conflicts.

**Pull request merged**